chef/cookbooks/bind9/recipes/default.rb

```diff
@@ -17,25 +17,25 @@
 require "ipaddr"
 
 package "bind9" do
-  case node[:platform]
-  when "centos","redhat", "suse"
+  case node[:platform_family]
+  when "rhel", "suse"
     package_name "bind"
   end
   action :install
 end
 package "bind9utils" do
-  case node[:platform]
-  when "centos","redhat", "suse"
+  case node[:platform_family]
+  when "rhel", "suse"
     package_name "bind-utils"
   end
   action :install
 end
 
-binduser, bindgroup = case node[:platform]
-  when "ubuntu","debian"
-    ["bind", "bind"]
-  when "centos","redhat","suse"
-    ["named", "named"]
+case node[:platform_family]
+when "debian"
+  binduser, bindgroup = ["bind", "bind"]
+when "rhel", "suse"
+  binduser, bindgroup = ["named", "named"]
```

Comment:
Style/ParallelAssignment: Do not use parallel assignment. (https://github.com/bbatsov/ruby-style-guide#parallel-assignment)